Lib/fontTools/cffLib/specializer.py

```diff
@@ -50,7 +50,10 @@ def programToCommands(program):
 				try:
 					width = stack.pop(0)
 				except IndexError:
-					raise ValueError(stack)
+					if ignoreErrors:
+						width = ''
```

Comment:
I understand your concern. I did it this way to not complicate and modify the code downstream.